src/commands/node/tasks.ts

```diff
@@ -1282,17 +1282,17 @@ export class NodeCommandTasks {
 
       const nodeId = Templates.nodeIdFromNodeAlias(config.nodeAlias) - 1;
       self.logger.info(`nodeId: ${nodeId}, config.newAccountNumber: ${config.newAccountNumber}`);
-      await self.accountManager.refreshNodeClient(config.namespace, config.nodeAlias);
-      config.nodeClient = await this.accountManager.loadNodeClient(config.namespace);
+      // await self.accountManager.refreshNodeClient(config.namespace, config.nodeAlias);
```

Comment:
this forces us to rebuild the node client with the newly updated node account ID  if the Node account ID was updated.